ui/src/models/trip.js

```diff
@@ -41,7 +41,13 @@ export default class Trip {
 
   price() {
     /** to do: return actual price */
-    return "$20";
+    let price = 0;
+    if (this.events.length > 0) {
+      this.events.map(event => {
+        price += parseInt(event.price);
```

Comment:
If `event.price` is not a number, ensure it is in the constructor for `Event` by putting `parseInt` there, if it isn't there already